internal/dataplane/translator/subtranslator/ingress_atc.go

```diff
@@ -26,10 +26,7 @@ var (
 	validHosts = regexp.MustCompile(`^(\*\.)?([a-zA-Z0-9]+(-[a-zA-Z0-9]+)*)+(\.([a-zA-Z0-9]+(-[a-zA-Z0-9]+)*))*?(\.\*)?$`)
 )
 
-var (
-	NormalIngressExpressionPriority RoutePriorityType = 1
-	IngressDefaultBackendPriority   RoutePriorityType = 0
-)
+var IngressDefaultBackendPriority RoutePriorityType = 0
```

Comment:
```suggestion
const IngressDefaultBackendPriority RoutePriorityType = 0
```
I think it should work 😉